test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -51,6 +55,45 @@ test_fail_tab_indentation:
     rules:
       indent_unit: tab
 
+# test_case_in_fn:
+#   fail_str: |
+#     SELECT
+#         MIN(MAX(COALESCE(
+#             a
+#         ))) AS c,
+#     MAX(CASE
+#             WHEN b THEN 1
+#             WHEN c THEN 1
+#         END) as agg
+#     FROM my_tbl
+#   fix_str: |
+#     SELECT
+#         MIN(MAX(COALESCE(
+#             a
+#         ))) AS c,
+#         MAX(CASE
+#             WHEN b THEN 1
+#             WHEN c THEN 1
+#         END) as agg
+#     FROM my_tbl
+
+test_fix_progressive_indentation_errors:
+  # Using tabs as indents works
```

Comment:
Are these actually tabs? If you're testing with tabs, I suggest using quoted strings with `\t` so someone's editor doesn't auto-convert them to spaces.

Reply:
yea can do. This wasnt my test not sure why its touched